### DISCUSSION TITLE
Weather Item not shown

### DISCUSSION THREAD

**Discussion by chris49a** [valinet/ExplorerPatcher#1492] [Q&A]
I want to change temperature units, but Weather is not listed in the property dialog.

What am I doing wrong?

![image](https://github.com/valinet/ExplorerPatcher/assets/42494277/ace6fe7c-da78-47ab-a654-46fbaef6bf67)

**Comment by pyrates999** [comment]
The setting between Fahrenheit and Celsius is a setting in windows: https://www.elevenforum.com/t/change-weather-widget-temperature-to-fahrenheit-or-celsius-in-windows-11.7197/

**Reply by chris49a** [reply]
Thanks, but that  doesn't solve my problem. It works fine for the Windows weather widget, but has no effect on the widget that seems to part of ExplorerPatcher, nor does it explain the missing Weather item in the original post.

![image](https://github.com/valinet/ExplorerPatcher/assets/42494277/7696a10d-9d5f-433e-a3b1-28d344051d30)

**Comment by pyrates999** [comment]
So the windows weather widget that is shown, while explorer patcher is installed, always shows it in Celsius?  No matter what you change it to in the settings for weather that windows provides?

**Reply by chris49a** [reply]
Windows app left, explorerplus right. Location and temperature for EP are wrong, too.

![image](https://github.com/valinet/ExplorerPatcher/assets/42494277/dcdbfbd6-9c2f-455c-b194-3621c13dd18f)

**Comment by pyrates999** [comment]
If the windows weather widget is working while explorer patcher is installed, then I would disable the weather feature in explorer patcher.
